### PR TITLE
DIV-6746: Solicitor is able to complete DN after alternative method or process server and AOS

### PR DESCRIPTION
**Pull request by madjava**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DIV-6746

### Change description ###
Updated the Solicitor screens when `Draft Decree Nisi Application` event is triggered to display alternative message when respondent answers documents are unavailable. This enables the process to move to the next step.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: test/unit/definitions/divorce/CaseEventToFields.test.js [hunk 1]
```diff
@@ -89,6 +91,7 @@ describe('CaseEventToFields (non-prod)', () => {
       'CaseEvent-deemed-and-dispensed-nonprod',
       'CaseEvent-general-email-nonprod',
       'CaseEvent-general-referral-nonprod',
+      'CaseEvent-alt-service-process-server-nonprod',
```
Comment (jvictorcf) [inline test/unit/definitions/divorce/CaseEventToFields.test.js:94]: Shouldn't you have both alternative service types here?

File: test/unit/definitions/divorce/CaseEventToFields.test.js [hunk 2]
```diff
@@ -98,7 +101,8 @@ describe('CaseEventToFields (non-prod)', () => {
       'CaseField-alt-service-process-server-nonprod',
       'CaseField-deemed-and-dispensed-nonprod',
       'CaseField-general-email-nonprod',
-      'CaseField-general-referral-nonprod'
+      'CaseField-general-referral-nonprod',
+      'CaseField-alternative-service-nonprod'
```
Comment (jvictorcf) [inline test/unit/definitions/divorce/CaseEventToFields.test.js:105]: Shouldn't you have both alternative service types here?

Comment (madjava) [reply test/unit/definitions/divorce/CaseEventToFields.test.js:105]: For alternative service its just `ServedByAlternativeMethod` being set to `Yes` or No` unlike Service Applications where we capture if its `Deemed` or `Dispensed`

**Review by Dizrupto — CHANGES_REQUESTED** [review]
You would need to run "yarn reset-ccd-module" to pull the changes for ccd-definition-processor to make sure there are no changes to the ccd-definition-processor

File: test/unit/definitions/divorce/CaseEventToFields.test.js (hunk 2 above)
Comment (Dizrupto) [reply test/unit/definitions/divorce/CaseEventToFields.test.js:105]: Both are there, I think the other one is at the top :)

Comment (yarpo) [reply test/unit/definitions/divorce/CaseEventToFields.test.js:105]: please try and keep the alphabetical order :)

**Comment by madjava** [comment]
> You would need to run "yarn reset-ccd-module" to pull the changes for ccd-definition-processor to make sure there are no changes to the ccd-definition-processor

@Dizrupto I have done that in resent commits. Doing it again produces no changes. Please advice

File: test/unit/definitions/divorce/CaseEventToFields.test.js (hunk 2 above)
Comment (madjava) [reply test/unit/definitions/divorce/CaseEventToFields.test.js]: @yarpo, names reorder.

File: definitions/divorce/json/CaseEventToFields/CaseEventToFields-alt-service-process-server-nonprod.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorApplyForDN",
+    "CaseFieldID": "ServedByProcessServer",
+    "PageFieldDisplayOrder": 3,
+    "DisplayContext": "READONLY",
+    "PageID": "dnreviewaos",
+    "PageLabel": "Review Acknowledgement of Service",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "DNApplyForDecreeNisi=\"NeverShow\"",
```
Comment (jvictorcf) [inline definitions/divorce/json/CaseEventToFields/CaseEventToFields-alt-service-process-server-nonprod.json:27]: Why does this field need to be here?

Comment (madjava) [reply definitions/divorce/json/CaseEventToFields/CaseEventToFields-alt-service-process-server-nonprod.json:27]: @jvictorcf Had same question originally, was getting errors in the build. @Dizrupto had explained its a CCD behaviour, and we need to fined this also. Do so and build went green.

**Review by Dizrupto — APPROVED** [review]

**Pull request merged**